assets/i18n/en.json

```diff
@@ -653,41 +663,56 @@
     "I18N_LEARNER_DASHBOARD_VIEW_SUGGESTION": "View Suggestion",
     "I18N_LEARNER_GROUPS_SECTION_TITLE": "Your Learner Groups",
     "I18N_LEARNER_GROUP_ADD_GROUP_DETAILS": "Add Group Details",
-    "I18N_LEARNER_GROUP_ADD_NEW_LEARNER": "New Learner",
     "I18N_LEARNER_GROUP_ADD_NEW_SYLLABUS_ITEMS": "Add New Syllabus Items",
     "I18N_LEARNER_GROUP_ADD_SYLLABUS_ITEMS": "Add Syllabus Items",
     "I18N_LEARNER_GROUP_ADD_TO_SYLLABUS": "Add to syllabus",
     "I18N_LEARNER_GROUP_ASSIGNED_SKILLS": "{skillsCount, plural, =1{Assigned Skill} other{Assigned Skills}}",
     "I18N_LEARNER_GROUP_ASSIGNED_STORIES": "{storiesCount, plural, =1{Assigned Story} other{Assigned Stories}}",
+    "I18N_LEARNER_GROUP_ASSIGNED_SYLLABUS_TAB": "Assigned Syllabus",
     "I18N_LEARNER_GROUP_BACK_TO_ALL_LEARNERS_PROGRESS": "Back To All Learners' Progress",
     "I18N_LEARNER_GROUP_BACK_TO_SYLLABUS": "Back To Syllabus",
     "I18N_LEARNER_GROUP_CREATED_TITLE": "Your group <b><[groupName]></b> has been created.",
     "I18N_LEARNER_GROUP_CREATION_NEXT_BUTTON_TEXT": "Next",
     "I18N_LEARNER_GROUP_CREATION_PREV_BUTTON_TEXT": "Previous Step",
     "I18N_LEARNER_GROUP_DETAILS_GROUP_DESCRIPTION": "Group Description (Describe the group objectives in 2-4 lines)",
     "I18N_LEARNER_GROUP_DETAILS_GROUP_TITLE": "Group Title",
+    "I18N_LEARNER_GROUP_DETAILS_MODAL_DESCRIPTION": "Group Description",
+    "I18N_LEARNER_GROUP_DETAILS_TITLE": "Details",
+    "I18N_LEARNER_GROUP_FACILITATOR_MODAL_TEXT": "Facilitator",
     "I18N_LEARNER_GROUP_GROUP_DETAILS_SECTION": "Group Details",
+    "I18N_LEARNER_GROUP_INVITATION_MODAL_HEADER": "Learner group invitation",
     "I18N_LEARNER_GROUP_INVITE_LEARNERS": "Invite Learners",
     "I18N_LEARNER_GROUP_INVITE_LEARNERS_BY_USERNAME": "Invite learners through username",
     "I18N_LEARNER_GROUP_INVITE_LEARNERS_PLACEHOLDER_TEXT": "Add username of the learner to invite and press enter",
```

Comment:
Can I see how this looks like? I'm not sure if such a detailed placeholder is necessary (am assuming this is a text field)

Reply:
Add the learner's username and hit enter

Reply:
Updated. Thanks @danapj !